scripts/simulate.js

```diff
@@ -109,14 +110,27 @@ async function fiatLeverage(amount) {
 
       if (daiEarned.gte(BigNumber.from(0))) {
         totalDaiEarned = totalDaiEarned.add(daiEarned);
+        totalFiatBorrowed = totalFiatBorrowed.add(receipt.fiatDebt);
         totalInterestPaid = totalInterestPaid.add(interestDai);
         daiBalance = receipt.daiBalance;
         gasDai = gasDai.add(receipt.gasDai);
         cycles++;
       }
-    }
+      else {
+        console.log(`Leverage loop finished
+        Fiat down to: ${receipt.effectiveFiatPrice}
+        Reserves to [${Math.round(hre.ethers.utils.formatUnits(receipt.reserves0,18))},${Math.round(hre.ethers.utils.formatUnits(receipt.reserves1,18))}]
+        FIAT makes up ${Math.round(receipt.reserves0.div(receipt.reserves0.add(receipt.reserves1))*100*100)/100}% of the reserves)}`)
+      }
+    } // end while over leverageCycle
   })();
 
+  const {fiatBalance,effectiveFiatPrice,reserves0,reserves1} = curveSwapDaiForFiat(totalFiatBorrowed)
+  console.log(`Paid back ${totalFiatBorrowed} in fiat. Current balance is ${fiatBalance}.
+    Effective price was: ${effectiveFiatPrice}.
```

Comment:
ya i hate that it basically copy-pastes, but in reverse. but it's useful to get the actual slippage paid. unless we assume it's just the same amount in reverse. but instead of trying to figure out how to account for that correctly, this does it for us.